MaterialDesignExtensions/Themes/ResourceDictionaryExtensions.cs

```diff
@@ -18,15 +18,38 @@ public static class ResourceDictionaryExtensions
     {
         private static readonly string MaterialDesignPrefix = "MaterialDesign";
 
-        private static readonly Guid CurrentThemeKey = Guid.NewGuid();
-        private static readonly Guid ThemeManagerKey = Guid.NewGuid();
+        // Changed to object so it can support both 4.0.0 and 3.2.0 versions of
+        // MaterialDesignThemes.Wpf.ResourceDictionaryExtensions class
+        private static readonly object CurrentThemeKey;
+        private static readonly object ThemeManagerKey;
```

Comment:
MaterialDesignThemes 4.0.0 changed the CurrentThemeKey to be a string instead of Guid. To support both Guid (3.2.0) and string (4.0.0) just using object. And these keys are used only as ResourceDictionary lookup keys. So this will not hurt performance.